bin/runtests

```diff
@@ -235,7 +235,7 @@ function init_dbal()
 
     write_info('Updating schema');
     $process = exec_sf_cmd(
-        'doctrine:schema:update --force'
+        'doctrine:schema:update --force --complete'
```

Comment:
fine currently. but there where issue in combination with doctrine migrations in the past